integration/util/scenario/chain_spec.js

```diff
@@ -34,8 +34,10 @@ async function baseChainSpec(validatorsInfoHash, tokensInfoHash, ctx) {
       eth_address: v.eth_account
   }));
 
-  let assets = tokens.all().map((token) => ({
-    asset: `Eth:${token.ethAddress()}`,
+  let assets = tokens.all()
+      .filter(token => token.symbol.toUpperCase() !== 'CASH')
```

Comment:
dev note - i am 99% sure we want this filter but before we had CASH in `SupportedAssets` storage

resulted in having to do some downstream fixes